src/main/java/com/amplitude/Constants.java

```diff
@@ -5,7 +5,7 @@
     String  API_URL                 = "https://api2.amplitude.com/2/httpapi";
     long    NETWORK_TIMEOUT_MILLIS  = 10000;
     String  SDK_LIBRARY             = "amplitude-java";
-    String  SDK_VERSION             = "1.0.0";
+    String  SDK_VERSION             = "1.1.0";
```

Comment:
You can auto generate this just like Android.

Reply:
BuildConfig is specific to Android runtime. I can't figure out a clean way to do it now. 

I saw one approach in Java where the Java file reads the Gradle file in:
```
Properties properties = new Properties()
properties.load(project.rootProject.file('local.properties').newDataInputStream())
def sdkDir = properties.getProperty('sdk.dir')
```

Reply:
https://stackoverflow.com/questions/46805846/how-do-you-specify-buildconfigfield-in-gradle-java-library-project-build-script We can do it later.